projects/igniteui-angular/src/lib/expansion-panel/expansion-panel.component.ts

```diff
@@ -121,9 +125,17 @@ export class IgxExpansionPanelComponent implements IgxExpansionPanelBase {
 
     toggle(evt?: Event) {
         if (this.collapsed) {
-            this.expand(evt);
+            this.open(evt);
         } else {
-            this.collapse(evt);
+            this.close(evt);
         }
     }
+
+    open(evt?: Event) {
```

Comment:
Are those supposed to be a part of a different PR?

Reply:
Yes, so that the expansion panel properly implements the IToggle interface.